ADD_SAMPLE_INTO_DEV_KIT.md

```diff
@@ -12,7 +12,10 @@ For new samples, use `dev` as the target branch for pull requests. The `dev` bra
 
 ## Testing
 
-TBD
+1. Download the [test package](./assets/dev-kit-test-package.vsix).
```

Comment:
We should give users a little more indication this link is to a file that'll download.
```suggestion
1. Download the [test package extension](./assets/dev-kit-test-package.vsix).
```